client/src/components/Ticketing/GroupeToggle.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
```

Comment:
Usually we just want to import as little as possible, so we probably want to do the specific things we're importing, even though I know thats what the mui docs look like

Reply:
On line 6, I use `const [filterAs, setFilter] = React.useState('Active');`. Is this still unneeded? I result in code being underlined in red.

Reply:
On line 6, I use ` const [filterAs, setFilter] = React.useState('Active');`. Is it still unneeded? Currently, removing the import results in code being underlined in red.

Reply:
Well, you wouldn't want to remove the import but something more along the lines of `import {useState} from 'react';`

Reply:
And that import statement would also need to include anything else that is imported from `'react'` like MouseEvent